src/components/CalendarMonth.jsx

```diff
@@ -104,7 +107,7 @@ export default class CalendarMonth extends React.Component {
     } = this.props;
 
     const { weeks } = this.state;
-    const monthTitle = month.format(monthFormat);
+    const monthTitle = renderMonth ? renderMonth(month) : month.format(monthFormat);
```

Comment:
hm, ok, fair point. this can stay as-is, and we can address them all holistically in the future.